src/coreclr/tools/aot/ILCompiler.Build.Tasks/ILCompiler.Build.Tasks.csproj

```diff
@@ -30,7 +30,7 @@
     <Content Include="$(NuGetPackageRoot)\system.reflection.metadata\$(SystemReflectionMetadataVersion)\lib\netstandard2.0\System.Reflection.Metadata.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="$(NuGetPackageRoot)\system.collections.immutable\$(SystemReflectionMetadataVersion)\lib\netstandard2.0\System.Collections.Immutable.dll">
+    <Content Include="$(NuGetPackageRoot)\system.collections.immutable\$(SystemCollectionsImmutableVersion)\lib\netstandard2.0\System.Collections.Immutable.dll">
```

Comment:
`SystemCollectionsImmutableVersion` does not exist anymore in eng\Versions.props (it was deleted recently). I think you need to add it back to fix the build breaks.